docs/source/install.ipynb

```diff
@@ -35,6 +35,11 @@
     "\n",
     "```\n",
     "\n",
+    "### Add-ons\n",
+    "**Update checker** - Receive automatic notifications of new Woodwork releases\n",
+    "```\n",
+    "python -m pip install woodwork[update_checker]\n",
```

Comment:
Use quotes so that you can install in all terminals (just like above)
```suggestion
    "python -m pip install \"woodwork[update_checker]\"\n",
```

Reply:
Do we want to use quotes in the README too?

Reply:
Yes we should